configure

```diff
@@ -296,9 +296,10 @@ install_local_rebar() {
         if [ ! -d "${rootdir}/src/rebar" ]; then
             git clone --depth 1 https://github.com/apache/couchdb-rebar.git ${rootdir}/src/rebar
         fi
-        make -C ${rootdir}/src/rebar
+        cd src/rebar
+        ./bootstrap
```

Comment:
Can you help me understand why it's preferable to run `bootstrap` explicitly when it appears that `make` [should be using bootstrap already?](https://github.com/apache/couchdb-rebar/blob/main/Makefile#L8-L9)

Reply:
I used the rebar3 example as a guide at the beginning.  I will test it again without this change.

Reply:
@jaydoane You are right, there are no advantages and it works without calling bootstrap directly. Closing this PR.